### ISSUE TITLE
Compatability issue with Python3.5

### ISSUE THREAD

**Issue by milesflo** [Axonius/axonius_api_client#69]
## Request Type
Bug

## Description
The API client has a compatibility issue with the 3.5 engine.. Notably the `strict` keyword in the pathlib definition described in the stack trace that follows

## Potential Fix
This may be out of scope for remediation as only 3.7< is supported according to the docs, but it may be worth addressing as 3.5 is still what ships in many distros.

## Stack Trace
```
Traceback (most recent call last):
  File "/opt/custom-analyzers/Axonius/axonius.py", line 4, in <module>
    import axonius_api_client as axonapi
  File "/usr/local/lib/python3.5/dist-packages/axonius_api_client/__init__.py", line 5, in <module>
    from . import api, auth, cli, constants, exceptions, http, logs, tools, version
  File "/usr/local/lib/python3.5/dist-packages/axonius_api_client/cli/__init__.py", line 8, in <module>
    from . import cli_constants, click_ext, context, grp_adapters, grp_objects, grp_tools
  File "/usr/local/lib/python3.5/dist-packages/axonius_api_client/cli/cli_constants.py", line 33, in <module>
    load_dotenv()
  File "/usr/local/lib/python3.5/dist-packages/axonius_api_client/cli/cli_constants.py", line 22, in load_dotenv
    ax_env_path = tools.path(obj=ax_env)
  File "/usr/local/lib/python3.5/dist-packages/axonius_api_client/tools.py", line 277, in path
    return pathlib.Path(obj).expanduser().resolve(strict=False)
TypeError: resolve() got an unexpected keyword argument 'strict'
```

**Comment by lifehackjim** [comment]
I can fix that easily enough - i'll add to next minor release

**Comment by lifehackjim** [comment]
Python 3.5 passed my test suite after quite a few modifications.

Please note, Python 3.5 is not in my standard pre-release test suite. But I'll fix issues if any are identified.

Changes will be published with the release of 2.2.0 by next week.

**Comment by milesflo** [comment]
Thanks Jim 👏
